homeassistant/components/iron_os/coordinator.py

```diff
@@ -22,6 +28,7 @@ class IronOSCoordinator(DataUpdateCoordinator[LiveDataResponse]):
     """IronOS coordinator."""
 
     device_info: DeviceInfoResponse
+    settings: SettingsDataResponse = {}
```

Comment:
I'm mostly worried that this is now a shared dict for all instances of the dataclass. What type is the settings data response?

Reply:
It is a TypedDict

Reply:
https://github.com/tr4nt0r/pynecil/blob/44ce391a5d8732e7e8628fb467ae863c34bd8c4c/pynecil/types.py#L226-L356

Reply:
I'm not sure how we should create one properly

Reply:
I think I can get rid of settings for now, as setpoint temp is also available in the livedata characteristic and I didn't include other settings characteristics yet...